azkaban-common/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
   compile('io.dropwizard.metrics:metrics-core:3.1.0')
   compile('io.dropwizard.metrics:metrics-jvm:3.1.0')
 
+  testCompile('org.apache.derby:derby:10.13.1.1')
```

Comment:
@HappyRay I'm only able to find 10.12.1.1 in our internal artifactory. Should we downgrade the version?